README.md

```diff
@@ -10,4 +10,14 @@ docker build --no-cache -f ./.docker/images/{APP}/Dockerfile -t localhost:5005/{
 docker-compose up --force-recreate --remove-orphans -V grafana app-server
 # NUM = 1..5
 docker-compose run k6 run /scripts/scenario{NUM}.js
+```
+
+## Pushing app image
+
+```shell
+docker login
+
+docker build --no-cache -f ./.docker/images/{language}/{app}/Dockerfile -t rarhaeu/mgr2022:{app}-app .
```

Comment:
no language dir in images directory -> just app directories :)

Reply:
fixed